realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -62,6 +54,18 @@
  * @see Realm#executeTransaction(Realm.Transaction)
  */
 public class RealmResults<E extends RealmModel> extends OrderedRealmCollectionImpl<E> {
+    public static <T extends RealmModel> RealmResults<T> getBacklinkResults(BaseRealm realm, Row row, Class<T> srcTableType, String srcFieldName) {
```

Comment:
Good call.  The proxies are in the `io.realm` project.  The one on `Collections` has to stay public, unfortunately, because it is in `io.realm.internal`
Fixed